docker-compose.yml

```diff
@@ -0,0 +1,64 @@
+version: '3.2'
+services:
+  kbs:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.grpc-as
+    # image: xynnn007/kbs:v0.5.0-rc2
+    command: [
+        "/usr/local/bin/kbs",
+        "--socket",
+        "0.0.0.0:8080",
+        "--config",
+        "/etc/kbs-config.json",
+        "--auth-public-key",
+        "/opt/confidential-containers/kbs/user-keys/public.pub",
+        "--insecure-http"
+      ]
+    restart: always # keep the server running
+    ports:
+      - "8080:8080"
+    volumes:
+      - ./data/kbs-storage:/opt/confidential-containers/kbs/repository:rw
+      - ./config/public.pub:/opt/confidential-containers/kbs/user-keys/public.pub
+      - ./config/kbs-config.json:/etc/kbs-config.json
+
+  as:
+    image: xynnn007/attestation-service:v0.5.0-rc0
+    ports:
+    - "50004:50004"
+    restart: always
+    volumes:
+    - ./data/attestation-service:/opt/confidential-containers/attestation-service:rw
+    command: [
```

Comment:
Do you think we should set a default AS config and provide the default config file in `kbs/config/as-config.json` as KBS config?

Reply:
Yes, added that